mllib/src/main/scala/org/apache/spark/ml/clustering/BisectingKMeans.scala

```diff
@@ -68,6 +68,12 @@ private[clustering] trait BisectingKMeansParams extends Params with HasMaxIter
     "The minimum number of points (if >= 1.0) or the minimum proportion " +
       "of points (if < 1.0) of a divisible cluster.", ParamValidators.gt(0.0))
 
+
+  setDefault(
```

Comment:
total nit: make these params in single line, like above places

Reply:
Fixed. Thanks!